BackEnd/src/main/java/com/exadel/carpoolfree/controller/PathController.java

```diff
@@ -0,0 +1,39 @@
+package com.exadel.carpoolfree.controller;
+
+
+import com.exadel.carpoolfree.model.Path;
+import com.exadel.carpoolfree.repository.PathRepository;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("api/path")
+public class PathController {
+    private final PathRepository pathRepository;
+
+    public PathController(PathRepository pathRepository) {
+        this.pathRepository = pathRepository;
+    }
+
+    @GetMapping("/{id}")
+    public String getCoordinatesById(final @PathVariable Long id) {
```

Comment:
Just to put it up, l I think it's not going to be a string here after we define the model for `Path`, for now it's ok

Reply:
done